_posts/2012-12-01-nsqd.md

```diff
@@ -137,14 +137,15 @@ can optionally listen on a third port for HTTPS.
   Query Params:
 
       topic - the topic to publish to
+      defer - for how many ms to delay message delivery
```

Comment:
Please mention that defer is optional.

I might not include "defer" in the example - it's not the typical way, or the simplest way, to use "/pub".